plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/DockerConnectorFacet.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*global Ext, NX*/
+
+/**
+ * @since 3.0
+ */
+Ext.define('NX.coreui.view.repository.facet.DockerConnectorFacet', {
+  extend: 'Ext.form.FieldContainer',
+  alias: 'widget.nx-coreui-repository-docker-connector-facet',
+  requires: [
+    'NX.I18n'
+  ],
+
+  /**
+   * @override
+   */
+  initComponent: function() {
+    var me = this;
+
+    me.items = [
+      {
+        xtype: 'fieldset',
+        cls: 'nx-form-section',
+        title: NX.I18n.get('Repository_Facet_DockerConnectorFacet_Title'),
+
+        items: [
+          {
+            xtype: 'fieldcontainer',
+            fieldLabel: NX.I18n.get('Repository_Facet_DockerConnectorFacet_HttpPort_FieldLabel'),
+            helpText: NX.I18n.get('Repository_Facet_DockerConnectorFacet_HttpPort_HelpText'),
+            layout: 'hbox',
+            items: [
+              me.createCheckbox('http'),
+              me.createPort('http')
+            ]
+          },
+          {
+            xtype: 'fieldcontainer',
+            fieldLabel: NX.I18n.get('Repository_Facet_DockerConnectorFacet_HttpsPort_FieldLabel'),
+            helpText: NX.I18n.get('Repository_Facet_DockerConnectorFacet_HttpsPort_HelpText'),
+            layout: 'hbox',
+            items: [
+              me.createCheckbox('https'),
+              me.createPort('https')
+            ]
+          }
+        ]
+      }
+    ];
+
+    me.callParent(arguments);
+  },
+
+  createCheckbox: function(type) {
+    return {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ASemicolon)